src/Integration.UnitTests/Binding/BindingProcessImplTests.cs

```diff
@@ -308,9 +305,6 @@ public async Task DownloadQualityProfile_WhenBindingConfigIsNull_Fails()
             result.Should().BeFalse();
             testSubject.InternalState.QualityProfiles[language].Should().Be(profile);
 
-            notifications.AssertProgress(0.0);
-            notifications.AssertProgressMessages(Strings.DownloadingQualityProfileProgressMessage);
-
```

Comment:
This test is checking what happens when a binding config is `null`. It doesn't need to check the progress messages too.